src/lender/definitions.sol

```diff
@@ -28,7 +28,7 @@ contract Definitions is FixedPoint, Math {
             currSeniorAsset, newReserve, nav), nav, newReserve);
     }
 
-/// calculates the net wealth in the system
+    /// calculates the net wealth in the system
```

Comment:
We can change the `///`  to `//`, we started this pattern but we are not consistent

Reply:
Good point! Changed.